src/EnergyPlus/OutputProcessor.hh

```diff
@@ -233,6 +233,7 @@ namespace OutputProcessor {
         J_m2,
         clo,
         W_K,
+        K_W,
```

Comment:
@JasonGlazer can you approve this addition?

Reply:
Yes. Looks fine.